whois-query/src/main/java/net/ripe/db/whois/query/dao/jdbc/JdbcAbuseValidationStatusDao.java

```diff
@@ -23,10 +24,10 @@ public JdbcAbuseValidationStatusDao(@Qualifier("internalsDataSource") final Data
     }
 
     @Override
-    public boolean isSuspect(CIString email) {
-        return internalsTemplate.queryForObject(
-                "select count(*) from abuse_email where address = ? and status = ?",
-                new String[] { email.toLowerCase(), "SUSPECT" },
+    public boolean isSuspect(@Nullable final CIString address) {
+        return (address != null) && internalsTemplate.queryForObject(
```

Comment:
why the null check?

Reply:
I added more input checks, but it's not necessary in this case, I removed it again